pkg/sysregistriesv2/system_registries_v2_test.go

```diff
@@ -159,3 +159,63 @@ unqualified-search = true`)
 	reg = FindRegistry("registry-c.com/foo:bar", unqRegs)
 	assert.NotNil(t, reg)
 }
+
+func TestV1BackwardsCompatibility(t *testing.T) {
```

Comment:
Maybe also preserve the original, ~unchanged `system_registries_test.go` as well?

Reply:
I extended the v2 ones a bit, which test more cases now, so I don't think we need that.